airbyte-integrations/connectors/source-intercom/source_intercom/source.py

```diff
@@ -54,7 +55,8 @@ def __init__(
 
         super().__init__(authenticator=authenticator)
 
-    def next_page_token(self, response: requests.Response) -> Optional[Mapping[str, Any]]:
+    @staticmethod
```

Comment:
The parent method is not static, how come this one is? I think it's a little clearer if this is not static. Took me a while to think through whether overriding a non static method with a static method is valid.

Reply:
Updated to not static.